src/main/scala/org/constellation/infrastructure/endpoints/SnapshotEndpoints.scala

```diff
@@ -172,8 +168,8 @@ class SnapshotEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F] {
 
   private def getLatestMajorityHeight(redownloadService: RedownloadService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
     case GET -> Root / "latestMajorityHeight" =>
-      (redownloadService.lowestMajorityHeight, redownloadService.latestMajorityHeight)
-        .mapN(LatestMajorityHeight(_, _))
+      redownloadService.getMajorityRange
+        .map(range => LatestMajorityHeight(range.from, range.to))
```

Comment:
what do you think about:
```scala
object LatestMajorityHeight {
  def apply(range: ...): LatestMajorityHeight = new LatestMajorityHeight(range.from, range.to)
}
```

and then:
```scala
.map(LatestMajorityHeight(_))
```
?

Reply:
Fine with me, I will change it.